test-expected/sequelize/db-analysis-output/default-values.expected.js

```diff
@@ -1,99 +0,0 @@
-const Sequelize = require('sequelize');
```

Comment:
[MISC] Did you commit the file in 2 times 🤔 . This file should just be marked as renamed

Reply:
I mark this as resolved, can be discussed in the comment about test-expected/sequelize/db-analysis-output/default_values.expected.js